cmd/edenStart.go

```diff
@@ -101,7 +104,12 @@ var startCmd = &cobra.Command{
 		if err := eden.StartEVEQemu(command, qemuARCH, qemuOS, eveImageFile, qemuSMBIOSSerial, qemuAccel, qemuConfigFile, eveLogFile, evePidFile); err != nil {
 			log.Errorf("cannot start eve: %s", err)
 		} else {
-			log.Infof("EVE is starting")
+			log.Infof("EVE is starting in QEMU")
+		}
+		if err := eden.StartEVEParallels(vmName, parallelsCpus, parallelsMem); err != nil {
```

Comment:
We start Qemu and then start Parallels as I can see from code. Is it expected behavior?